<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2094</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="infoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>771</width>
     <height>378</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_35">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="avatarLayout"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_33">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_49">
                <item>
                 <widget class="QLabel" name="label_76">
                  <property name="text">
                   <string>Фамилия</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_77">
                  <property name="text">
                   <string>Имя</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_50">
                <item>
                 <widget class="QLineEdit" name="lineName"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineLastName"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Дата рождения</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Родной город</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="lineBD"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineHomeTown"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_52">
                <item>
                 <widget class="QLabel" name="label_79">
                  <property name="text">
                   <string>Город</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_80">
                  <property name="text">
                   <string>Id</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_78">
                  <property name="text">
                   <string>Страна</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_51">
                <item>
                 <widget class="QLineEdit" name="lineCity"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineCityId"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineCountry"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="lineStatus">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QPushButton" name="btn_save">
                  <property name="text">
                   <string>Сохранить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_setting_privacy">
                  <property name="maximumSize">
                   <size>
                    <width>170</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Настройки приватности</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="countFriends">
                <property name="contextMenuPolicy">
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="lineWidth">
                 <number>3</number>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::PlainText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_friends">
                <property name="text">
                 <string>Друзья</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="countGroups">
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="btn_groups">
                  <property name="text">
                   <string>Группы</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_add_group">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_clearingGroups">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="countPhoto">
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QPushButton" name="btn_photo">
                  <property name="text">
                   <string>Фото</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_addPhoto">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="photosLayout">
            <item>
             <widget class="QLabel" name="labelImg1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelImg2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelImg3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelImg4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
